browser/themes/brave_dark_mode_utils.cc

```diff
@@ -28,11 +28,6 @@ namespace {
 bool g_is_test_ = false;
 bool g_system_dark_mode_enabled_in_test_ = false;
 
-void ClearBraveDarkModeProfilePrefs(PrefService* prefs) {
-  prefs->ClearPref(kBraveThemeType);
```

Comment:
Let's delete `kBraveThemeType` and `kUseOverriddenBraveThemeType` vars from `components/constants/pref_names.h` together. 👍🏼

Reply:
Fixed in https://github.com/brave/brave-core/pull/21524/commits/1ac1432c6269c62ef9519d2a74448779b2f7c781.